src/stories/TextInput.stories.js

```diff
@@ -57,6 +57,70 @@ Placeholder.args = {
     placeholder: 'My placeholder text',
 };
 
+const AutoGrow = Template.bind({});
+AutoGrow.storyName = 'Autogrow input';
+AutoGrow.args = {
+    label: 'Autogrow input',
+    name: 'AutoGrow',
+    placeholder: 'My placeholder',
+    autoGrow: true,
+    textInputContainerStyles: [{
+        minWidth: 150,
+    }],
+};
+
+const Prefixed = Template.bind({});
+Prefixed.storyName = 'Prefixed input';
+Prefixed.args = {
+    label: 'Prefixed input',
+    name: 'Prefixed',
+    placeholder: 'My placeholder text',
+    prefixCharacter: '@',
+};
+
+const WithFixedLabel = Template.bind({});
+WithFixedLabel.storyName = 'Always active label';
```

Comment:
Why is this named differently than the component name? Readability? I think ideally we'd keep them as consistent as possible.